packages/@coorpacademy-components/src/molecule/setup-slide/setup-cohort-item-popin.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import style from './popinstyle.css';
+
+const SetupCohortItemPopin = props => {
```

Comment:
this is too specific, please move this sub component as `molecule.messagePopin` so it may easily be reused elsewhere

Reply:
It' done